packages/caravan-clients/src/bitcoind.ts

```
@@ -102,32 +124,51 @@ export async function bitcoindGetAddressStatus({ url, auth, address }) {
 export async function bitcoindSendRawTransaction({ url, auth, hex }) {
   try {
-    const resp = await callBitcoind(url, auth, "sendrawtransaction", [hex]);
+    const resp = await callBitcoind<string>(url, auth, "sendrawtransaction", [
+      hex,
+    ]);
     return resp.result;
-  } catch (e) {
+  } catch (e: any) {
     // eslint-disable-next-line no-console
     console.log("send tx error", e);
     throw (e.response && e.response.data.error.message) || e;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)